package.json

```diff
@@ -48,11 +48,12 @@
     "webpack:prod": "lerna run webpack:prod",
     "webpack:prod:web": "lerna run webpack:prod:web",
     "webpack:prod:node": "lerna run webpack:prod:node",
-    "test:all": "tap --ts --node-arg=--max-old-space-size=4096 --timeout=3600 --branches=45 --functions=70 --lines=75 --statements=75 \"packages/cactus-*/src/test/typescript/{unit,integration,benchmark}/\"",
-    "test:unit": "tap --ts --node-arg=--max-old-space-size=4096 --timeout=600 --no-check-coverage \"packages/cactus-*/src/test/typescript/unit/\"",
+    "test:all": "tap --ts --node-arg=--max-old-space-size=4096 --timeout=3600 --branches=45 --functions=70 --lines=75 --statements=75 \"packages/cactus-*/src/test/typescript/{unit,integration,benchmark}/\" ",
+    "test:unit": "tap --ts --node-arg=--max-old-space-size=4096 --timeout=600 \"packages/cactus-*/src/test/typescript/unit/\" ",
```

Comment:
@travis-payne I think we still need the coverage check in the CI as well, unless the badge has some feature that makes it so that the PRs are blocked via failing checks when the coverage is below threshold (don't know, I'm asking)

Reply:
Don't believe it does, I'll revert this.

Reply:
@travis-payne Cheers!